src/main/features/eligibility/store.ts

```diff
@@ -1,18 +1,14 @@
-import * as config from 'config'
 import * as express from 'express'
-import * as Cookies from 'cookies'
 import * as _ from 'lodash'
 
 import { Eligibility } from 'eligibility/model/eligibility'
+import { CookieProperties } from 'shared/cookieProperties'
 
-const minuteInMilliseconds = 60 * 1000
-const cookieTimeToLiveInMinutes = config.get<number>('eligibility.cookie.timeToLiveInMinutes') * minuteInMilliseconds
-
-export const cookieName = 'eligibility-check'
+export const cookieName = 'eligibilityCheck'
 
 export class CookieEligibilityStore {
   read (req: express.Request, res: express.Response): Eligibility {
-    const cookie: string = new Cookies(req, res).get(cookieName)
+    const cookie: string = req.signedCookies.eligibilityCheck
```

Comment:
will this just auto encrypt cookies that aren't encrypted ? or just make those users restart, chances are it'll be really low number of people that could be affected?

Reply:
it didn't like the - in the cookie name so I had to rename it. Yeah it should hopefully only effect a very small number of users if any at all.

Reply:
cookies page probably needs an update if the cookies been renamed

Reply:
Good spot! done.